dist/Module-CoreList/corelist

```diff
@@ -498,11 +499,11 @@ sub format_perl_version {
 sub numify_version {
     my $ver = shift;
     if ($ver =~ /\..+\./) {
-	have_version_pm()
-	    or die "You need to install version.pm to use dotted version numbers\n";
+        have_version_pm()
```

Comment:
This commit seems to contain a lot of unrelated whitespace changes

Reply:
@Leont not so many actually just what I saw when was fixing the bug. Nevertheless that's not good?

Reply:
Doing that in the same commit where you're making semantic changes is always bad.

Reply:
@Leont Thank you. I will know for the future changes.